index.html

```diff
@@ -27,6 +27,11 @@
       shape-rendering: crispEdges;
     }
 
+    .axis .minor line {
+      stroke: #777;
```

Comment:
Note that this makes the "minor" gridline darker than the "major" gridline (which uses `#ccc`). Be careful when reusing code from Blocks that you don't accidentally bring in unwanted side effects.